dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -764,4 +764,77 @@ dotnet_diagnostic.ACL1014.allowed_words = B2C,365
 ```
 Note the following:
 - These overrides are case-insensitive. For example. an identifier with 'b2c' in its name would not violate this rule.
-- These overrides can contain letters. For example, an identifier with '2' in its name would violate this rule.
\ No newline at end of file
+- These overrides can contain letters. For example, an identifier with '2' in its name would violate this rule.
+
+## ACL1015 - Controller actions does not have ProducesResponseType attribute when return type is TypedResults
```

Comment:
I think the point of this analyzer is that the `ProducesResponseType` attribute doesn't add value if we're using typed results. Therefore, this warning should be applied if it has the ProducesResponseType attribute, not when it doesn't - does that make sense?